geode-core/src/test/java/org/apache/geode/management/internal/operation/OperationHistoryManagerTest.java

```diff
@@ -83,9 +90,13 @@ public void getLotsOfCompleted() {
     future2.complete(null);
     CompletableFuture<JsonSerializable> future3 = new CompletableFuture<>();
     history.save(op("3", future3));
```

Comment:
It seems like having multiple items in this unit test is now just noise since we don't have an LRU limit. I think you want a unit test that shows that both save and getOperationInstance will expire things. I think it would be good to add another method, just for the unit test, to OperationHistoryManager that lets you get an operation instance without calling expire. That allows the unit test to look without causing a change

Reply:
Instead I moved the call to System.currentTimeMillis into a `now()` method that can be mocked to give tests "virtual time" ability.